statistics/handle/globalstats/global_stats.go

```diff
@@ -54,12 +54,24 @@ type GlobalStats struct {
 	ModifyCount           int64
 }
 
+func newGlobalStats(histIDsLen int) *GlobalStats {
```

Comment:
I think the var name like `histCount` will be better than this one?

Reply:
Good point! Addressed!